systems/subscriber/sim-manager/go.mod

```diff
@@ -6,7 +6,7 @@ replace github.com/ukama/ukama/systems/common => ../../common
 
 replace github.com/ukama/ukama/systems/init/msgClient => ../../init/msgClient
 
-replace github.com/ukama/ukama/systems/data-plan/package => ../../data-plan/package
+// replace github.com/ukama/ukama/systems/data-plan/package => ../../data-plan/package
```

Comment:
If this line is no more needed, then we can remove it.

Reply:
addressed